spec/rails_helper.rb

```diff
@@ -1,31 +1,19 @@
-# This file is copied to spec/ when you run "rails generate rspec:install"
 ENV["RAILS_ENV"] ||= "test"
 require "spec_helper"
 require File.expand_path("../../config/environment", __FILE__)
 require "rspec/rails"
+require 'capybara/poltergeist'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.